image/auto_build.rb

```diff
@@ -96,5 +96,5 @@ def dev_deps()
   puts "Building #{images[image]}"
   dev_deps() if image == :discourse_dev
 
-  build(images[image])
+  build(images[image], ARGV[1..-1].join(" "))
```

Comment:
This is to allow us to do `ruby auto_build.rb base_slim --build-args="RUBY_VERSION=3.3.1"`